code/modules/mob/hearing/virtualhearer.dm

```diff
@@ -71,12 +71,12 @@ var/list/stationary_hearers = list(	/obj/item/device/radio/intercom,
 		sight = copying
 	if(adding)
 		sight |= adding
-		if(adding & SEE_TURFS)
+		if((adding & SEE_TURFS) || (adding & SEE_MOBS))
 			sight &= ~SEE_BLACKNESS
 	if(removing)
 		sight &= ~removing
-		if(removing & SEE_TURFS)
-			sight |= SEE_BLACKNESS
+		if((removing & SEE_TURFS) || (removing & SEE_MOBS))
+			sight |= SEE_BLACKNESS 
```

Comment:
```suggestion
			sight |= SEE_BLACKNESS
```
This was just some weird trailing whitespace your text editor didn't remove for some reason